src/wham_v0.cpp

```diff
@@ -157,14 +211,33 @@ Type objective_function<Type>::operator() ()
   PARAMETER_MATRIX(sel_repars);    // fixed effect parameters controlling selpars_re, dim = n_blocks, 3 (sigma, rho, rho_y)
   //PARAMETER_VECTOR(catch_paa_pars);
   PARAMETER_MATRIX(catch_paa_pars); //n_fleets x 3
+  PARAMETER_MATRIX(catch_pal_pars); //n_fleets x 3 
   //PARAMETER_VECTOR(index_paa_pars);
   PARAMETER_MATRIX(index_paa_pars); //n_indices x 3
+  PARAMETER_MATRIX(index_pal_pars); //n_indices x 3 
   PARAMETER_VECTOR(M_a); // mean M-at-age, fixed effects, length = n_ages if M_model = 2 (age-specific), length = 1 if M_model = 1 (constant) or 3 (weight-at-age M)
   PARAMETER_ARRAY(M_re); // random effects for year- and age-varying M deviations from mean M_a), dim = n_years x n_M_a
   PARAMETER_VECTOR(M_repars); // parameters controlling M_re, length = 3 (sigma_M, rho_M_a, rho_M_y)
+  
+  // section: growth parameters:
+  PARAMETER_MATRIX(growth_a);  
+  PARAMETER_ARRAY(growth_re);    
+  PARAMETER_MATRIX(growth_repars);    
+  PARAMETER_VECTOR(LAA_a); 
+  PARAMETER_ARRAY(LAA_re);  
+  PARAMETER_VECTOR(LAA_repars); 
+  PARAMETER_VECTOR(SD_par);
```

Comment:
I think SD_par should be renamed to something clearer like LAA_SD_pars

Reply:
renamed